jobrunner/local_run.py

```diff
@@ -77,20 +77,38 @@ def add_arguments(parser):
         help="Don't stop on first failed action",
         action="store_true",
     )
+    parser.add_argument(
+        "--debug",
```

Comment:
I feel `--debug` is slightly overloaded - implies "log verbosely" - but I can't think of a better alternative!

Reply:
It is a bit. But I sort of imagine this flag will increase log verbosity as well at some point when I get round to it.